zqd/core.go

```diff
@@ -33,7 +25,10 @@ type Core struct {
 	logger       *zap.Logger
 }
 
-func NewCore(conf Config) (*Core, error) {
+var Version string
+
+func NewCore(conf Config, version string) (*Core, error) {
+	Version = version
```

Comment:
Won't this cause weird memory race issues, notably in highly concurrent unit tests? Consider adding version as a value in zqd.Config.

Reply:
Oops.  Yes, of course.  Will do.